src/assets/movies/movies.ts

```diff
@@ -0,0 +1,59 @@
+import avengers from "../images/avengers.png";
```

Comment:
This file contains hardcoded data will be better store this file in mocked folder as well as these imported images